code/datums/supplypacks.dm

```diff
@@ -326,20 +326,65 @@ var/list/all_supply_groups = list(supply_emergency,supply_security,supply_engine
 /////// Weapons: Specialist
 
 /datum/supply_packs/security/armory/ballistic
+	name = "Riot Shotguns Crate"
+	contains = list(/obj/item/weapon/gun/projectile/shotgun/riot,
+					/obj/item/weapon/gun/projectile/shotgun/riot,
+					/obj/item/weapon/gun/projectile/shotgun/riot,
```

Comment:
I'm not disagreeing shotguns should stay somewhat rare, but I have to say these are probably going to be as rare as combat shotguns are now if not more from the cost. If you really think about it, there is almost no reason all officers don't carry around an auto-rifle or have a combat shotgun other then SoP and the fact no-one buys them until they are needed.